compiler/test/input/wasiPolyfill.gr

```diff
@@ -0,0 +1,7 @@
+import foreign wasm fd_write : (WasmI32, WasmI32, WasmI32, WasmI32) -> WasmI32 from "wasi_snapshot_preview1"
+
+export let fd_write: (WasmI32, WasmI32, WasmI32, WasmI32) -> WasmI32 = (fd, iovs, iovs_len, nwritten) => {
+    fd_write(fd, iovs, iovs_len, nwritten)
+    fd_write(fd, iovs, iovs_len, nwritten)
```

Comment:
Why do you call fd_write three times?

Reply:
Ah, to test this is the polyfill being called, and not the real one right?

Reply:
Yup!